runtime/master/src/main/java/org/apache/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -313,11 +314,15 @@ public void requestContainer(final String resourceSpecificationString) {
           final TreeNode resourceNode = jsonRootNode.get(i);
           final String type = resourceNode.get("type").traverse().nextTextValue();
           final int memory = resourceNode.get("memory_mb").traverse().getIntValue();
+          final double maxOffheapRatio =
+            (resourceNode.path("max_offheap_ratio").traverse().nextToken() == JsonToken.VALUE_NUMBER_FLOAT)
+              ? resourceNode.path("max_offheap_ratio").traverse().getFloatValue() : 0.02;
```

Comment:
Same here too. I see no good rationales on setting the default value to `0.02`.

Reply:
If this value is not explicitly set, I think it'd be good to set this value to `0`.